packages/server/src/server.ts

```diff
@@ -208,14 +180,7 @@ export class ProsopoServer {
             // We don't care about recency with PoW challenges as they are single use, so just return the verified result
             return result.verified
         }
-        const recent = await verifyRecency((await this.getContractApi()).api, blockNumber, timeouts.image.cachedTimeout)
-
-        if (!recent) {
-            // bail early if the block is too old. This saves us calling the Provider.
```

Comment:
It's still worthwhile putting the equivalent check in here for signed timestamp, to avoid unnecessary calls to the provider.